assets/runtime/functions

```diff
@@ -858,7 +858,8 @@ gitlab_configure_secrets() {
   update_template ${GITLAB_SECRETS_CONFIG} \
   GITLAB_SECRETS_DB_KEY_BASE \
   GITLAB_SECRETS_SECRET_KEY_BASE \
-  GITLAB_SECRETS_OTP_KEY_BASE
+  GITLAB_SECRETS_OTP_KEY_BASE \
+  GITLAB_SECRETS_ENCRYPTED_SETTINGS_KEY_BASE
```

Comment:
Is it problematic if `GITLAB_SECRETS_ENCRYPTED_SETTINGS_KEY_BASE` is not set? 

For all other variables there is an explicit check to avoid empty strings. Should this check applied to `GITLAB_SECRETS_ENCRYPTED_SETTINGS_KEY_BASE` as well?

Reply:
Hi @sachilles,

Sorry for the delay, I was not notified of your mention.

Well gitlab starts, restarts and upgrades correctly without it within the current image. As long as you do not set it, it is ok.

But:
- if you set it on a newly created instance, you need to keep it to make sure the secret is not overriden and to be able to decode already stored encrypted settings;
- if you set it on an existing instance, you need to make sure to use the existing value of `encrypted_settings_key_base` inside `/home/git/gitlab/config/secrets.yml` if it exists, of not to set it if it is empty unless you never used encrypted settings.

By default this option is not set, and can be ignored. Thus it seems to me it should not warn neither block the user if it is not set (cf. https://docs.gitlab.com/ee/administration/encrypted_configuration.html)